### DISCUSSION TITLE
:pray:  Please provide release version policy :calendar:

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#6491] [Ideas]
Hi,

do you have a version/support release policy ? Would you share some elements on the repo ?
That would be very useful for the maintenance process. :smile_cat: :pray:

**Comment by hannes** [comment]
What would you expect there on why?

**Reply by adriens** [reply]
Hi @hannes , thank you for the feedback. In fact I'd like to add duckdb to [`endoflife.date`](https://endoflife.date/#contributing) as we'll use it on our stack. Would you be intersted in contributing duckdb to [`endoflife.date`](https://github.com/endoflife-date/endoflife.date/) :grey_question:

**Reply by hannes** [reply]
Interesting, but EOL might be a bit premature given that we're currently in pre-release?

**Reply by adriens** [reply]
Not sure as duckdb may have a fast growing community. For example I'm using it and will very probably adopt it of internal purpose... event while it's a pre-release :smile:

**Reply by adriens** [reply]
You could for example say that only the latest version is actually supportted :thought_balloon: ... or juste create a frst ajor cycle `0`
